lib/ES/Template.pm

```diff
@@ -73,6 +76,19 @@ my $Autosense_RE = qr{
         :(?:CONSOLE|AUTOSENSE|KIBANA):
     }xs;
 
+#===================================
+sub _build_js {
+#===================================
+    my ( $self ) = (@_);
+
+    if ($self->{compiled_JS} == 0) {
+        print "Building docs.js\n";
+        `node_modules/parcel/bin/cli.js build resources/web/docs_js/index.js --no-minify -d resources/web -o docs.js`;
```

Comment:
I'd prefer you use the `run` method from `Util.pm` here so it'll log its output in the same way as all of the other subprocesses.

Reply:
Ah right, I have that written down from our convo, thanks for the reminder.